services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/activities_progress_report.jsx

```diff
@@ -116,7 +116,12 @@ export default createReactClass({
         Header: 'Student',
         accessor: 'student_id',
         resizeable: false,
-        Cell: ({ row }) => <span className={cellClassName}>{studentFilters.find(student => student.value == row.original.student_id)?.name}</span>,
+        Cell: ({ row }) => {
+          const { original } = row
+          const { id, name, link } = original
+          const label = studentFilters.find(student => student.value == row.original.student_id)?.name
```

Comment:
Should we just extract `student_id` as a const above, and then make this `find` call just compare `student.value == student_id`?

Reply:
yes, I'll make those extractions I missed